scripts/userstyles.yml

```diff
@@ -427,7 +427,7 @@ userstyles:
     readme:
       usage: "Switch to a default Lichess light/dark theme via **Username** > **Background** for the best experience!"
       app-link: "https://lichess.org/"
-      current-maintainers: [*coopw1]
+      current-maintainers: [*coopw1, *kingofcube]
```

Comment:
you also need to add your name to the contributes list at the top of the file, see the first step of https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md#userstylesyml

Reply:
mkay